src/shared/cgroup-show.c

```diff
@@ -310,3 +316,52 @@ int show_cgroup_and_extra_by_spec(
 
         return show_cgroup_and_extra(controller, path, prefix, n_columns, extra_pids, n_extra_pids, flags);
 }
+
+int show_cgroup_get_root(
```

Comment:
hmm, I am not a fan of library calls that log on their own. We have some of those, but I try to call them "_and_warn" or so, to indicate that they are just shared code, but not really library calls, if you follow what I mean.

Hence maybe call this show_cgroup_get_root_and_warn()? or so?